tests/invariants/fail/constructor_indirect_assignment.act

```diff
@@ -0,0 +1,11 @@
+constructor of LValue
+interface constructor()
+
+creates
+
+	uint y := x
+	uint x := 2
+
+invariants
+
+	y == 2
```

Comment:
This test should pass. Added to fail so that this passes for now, and fails when it's fixed.